src/components/application_manager/include/application_manager/application_impl.h

```diff
@@ -188,9 +190,18 @@ class ApplicationImpl : public virtual InitialApplicationDataImpl,
   DataAccessor<VehicleInfoSubscriptions> SubscribedIVI() const OVERRIDE;
   inline bool IsRegistered() const OVERRIDE;
 
-  /**
-   * @brief ResetDataInNone reset data counters in NONE
-   */
+#ifdef SDL_REMOTE_CONTROL
+  bool SubscribeToInteriorVehicleData(
```

Comment:
descriptions for the new added functions?

Reply:
Confirmed Resolved 👍